upstream-sha256: bec10852a36108d8e17b2c970180c2b25f0fe33f34bb06d4a75a72e0fc2122da
actual-sha256: 0ecda6eb7ba4816603923403d0bd486dc92d7760ccb19a94c803c377bfbf8b84
--- upstream.expected
+++ tolk.actual
@@ -6,17 +6,6 @@
   int lidx0 = threadIdx.x; /* 16 */
   int alu0 = (lidx0<<2);
   float4 val0 = (*((float4*)((data1_128+alu0))));
-  *((float4*)((data0_64+alu0))) = val0;
-}
----
-typedef unsigned int uint;
-#define INFINITY (__int_as_float(0x7f800000))
-#define NAN (__int_as_float(0x7fffffff))
-template <class T, class F> __device__ __forceinline__ T tg_bitcast(F v) { union U { F f; T t; }; U u; u.f = v; return u.t; }
-extern "C" __global__ void __launch_bounds__(16) E_16_4(float* data0_64, float* data1_64) {
-  int lidx0 = threadIdx.x; /* 16 */
-  int alu0 = (lidx0<<2);
-  float4 val0 = (*((float4*)((data1_64+alu0))));
   *((float4*)((data0_64+alu0))) = val0;
 }
 ---
@@ -35,32 +24,10 @@
 #define INFINITY (__int_as_float(0x7f800000))
 #define NAN (__int_as_float(0x7fffffff))
 template <class T, class F> __device__ __forceinline__ T tg_bitcast(F v) { union U { F f; T t; }; U u; u.f = v; return u.t; }
-extern "C" __global__ void __launch_bounds__(16) E_16_4(float* data0_64, float* data1_64) {
-  int lidx0 = threadIdx.x; /* 16 */
-  int alu0 = (lidx0<<2);
-  float4 val0 = (*((float4*)((data1_64+alu0))));
-  *((float4*)((data0_64+alu0))) = val0;
-}
----
-typedef unsigned int uint;
-#define INFINITY (__int_as_float(0x7f800000))
-#define NAN (__int_as_float(0x7fffffff))
-template <class T, class F> __device__ __forceinline__ T tg_bitcast(F v) { union U { F f; T t; }; U u; u.f = v; return u.t; }
 extern "C" __global__ void __launch_bounds__(16) E_16_4(float* data0_64, float* data1_128) {
   int lidx0 = threadIdx.x; /* 16 */
   int alu0 = (lidx0<<2);
   float4 val0 = (*((float4*)((data1_128+alu0))));
-  *((float4*)((data0_64+alu0))) = val0;
-}
----
-typedef unsigned int uint;
-#define INFINITY (__int_as_float(0x7f800000))
-#define NAN (__int_as_float(0x7fffffff))
-template <class T, class F> __device__ __forceinline__ T tg_bitcast(F v) { union U { F f; T t; }; U u; u.f = v; return u.t; }
-extern "C" __global__ void __launch_bounds__(16) E_16_4(float* data0_64, float* data1_64) {
-  int lidx0 = threadIdx.x; /* 16 */
-  int alu0 = (lidx0<<2);
-  float4 val0 = (*((float4*)((data1_64+alu0))));
   *((float4*)((data0_64+alu0))) = val0;
 }
 ---
@@ -79,17 +46,6 @@
 #define INFINITY (__int_as_float(0x7f800000))
 #define NAN (__int_as_float(0x7fffffff))
 template <class T, class F> __device__ __forceinline__ T tg_bitcast(F v) { union U { F f; T t; }; U u; u.f = v; return u.t; }
-extern "C" __global__ void __launch_bounds__(16) E_16_4(float* data0_64, float* data1_64) {
-  int lidx0 = threadIdx.x; /* 16 */
-  int alu0 = (lidx0<<2);
-  float4 val0 = (*((float4*)((data1_64+alu0))));
-  *((float4*)((data0_64+alu0))) = val0;
-}
----
-typedef unsigned int uint;
-#define INFINITY (__int_as_float(0x7f800000))
-#define NAN (__int_as_float(0x7fffffff))
-template <class T, class F> __device__ __forceinline__ T tg_bitcast(F v) { union U { F f; T t; }; U u; u.f = v; return u.t; }
 extern "C" __global__ void __launch_bounds__(16) E_16_4(float* data0_64, float* data1_64, float* data2_64) {
   int lidx0 = threadIdx.x; /* 16 */
   int alu0 = (lidx0<<2);
